How to build from source

git cloned repo, did npm install, ran npm run build, get lots of errors. Is there a different build process?

```
> rete@1.4.2 lint /rete
> tsc && eslint --ext .ts --ext .js src


/rete/node_modules/ts-node/src/index.ts:240
    return new TSError(diagnosticText, diagnosticCodes)
           ^
TSError: ⨯ Unable to compile TypeScript:
src/index.js:1:1 - error TS1208: Cannot compile namespaces when the '--isolatedModules' flag is provided.

1 "use strict";
  ~~~~~~~~~~~~

    at createTSError (/rete/node_modules/ts-node/src/index.ts:240:12)
    at reportTSError (/rete/node_modules/ts-node/src/index.ts:244:19)
    at getOutput (/rete/node_modules/ts-node/src/index.ts:360:34)

```

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -2,7 +2,8 @@
     "extends": "rete-cli/tsconfig.json",
     "compilerOptions": {
         "target": "es5",
-        "downlevelIteration": true
+        "downlevelIteration": true,
+	"isolatedModules": false
     },
     "include": ["src"]
 }

